src/pages/Governance/styles.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useContext } from 'react'
+import styled, { ThemeContext } from 'styled-components'
+import Card from '../../components/Card'
+import { AutoRow } from '../../components/Row'
+import { Text } from 'rebass'
+import CurrencyLogo from '../../components/CurrencyLogo'
+import DoubleCurrencyLogo from '../../components/DoubleLogo'
+import { transparentize } from 'polished'
+
+const LightCardWrap = styled(Card)`
+  border: 1px solid ${({ theme }) => transparentize(0.3, theme.bg2)};
+  background-color: ${({ theme }) => transparentize(0.3, theme.bg1)};
+  padding: 0.8rem;
+  width: calc(25% - 6px);
+  height: 96px;
+  display: flex;
+  flex-wrap: wrap;
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+    width: calc(33% - 4px);
+  `};
+  ${({ theme }) => theme.mediaWidth.upToExtraSmall`
+    width: calc(50% - 4px);
+  `};
+`
+
+const StyledText = styled(Text)`
+  text-wrap: none;
+`
+export const GovernanceCard = () => {
+  const theme = useContext(ThemeContext)
+  const shittyCounter = 2
```

Comment:
Overlooking the name of the variable (ahaha) I don't necessarily understand why this is here. I see that it's hardcoded to 2, and then there are some checks that look to be not necessary at all below here?

Reply:
yeah that was for testing purposes just so we can see how it would work for different cards but i'm deleting it!